packages/blade-mcp/cursorRules/frontend-blade-rules.mdc

```diff
@@ -2,7 +2,7 @@
 description: 
 globs: 
 alwaysApply: true
-rules_version: <!-- dynamic_version -->
+rules_version: 0.0.8
```

Comment:
so we are now directly downloading this file using curl so that's why this rule is hard coded